sdk/python/kfp/_client.py

```diff
@@ -96,12 +98,14 @@ def __init__(self, host=None, client_id=None, namespace='kubeflow'):
 
   def _load_config(self, host, client_id, namespace):
     config = kfp_server_api.configuration.Configuration()
-    host = host or os.environ.get(KF_PIPELINES_ENDPOINT_ENV)
```

Comment:
Is this env variable no longer used?

Reply:
Could you help cross check the caller. the original code looks a bit funky to me.

Reply:
There was a person who wanted to set entrypoints using env variables (he was probably using shell scripts for automation.).

Reply:
that is fine. i pass in self._uihost in caller. should be equavalent.

Reply:
the line here is contradict to how we get host in line 90 so can potentially be buggy.

Reply:
`uihost` is read from `KF_PIPELINES_UI_ENDPOINT_ENV` while the host is read from `KF_PIPELINES_ENDPOINT_ENV`

Reply:
Maybe you just move the `host = host or os.environ.get(KF_PIPELINES_ENDPOINT_ENV)` up into `__init__`?
```python
host = host or os.environ.get(KF_PIPELINES_ENDPOINT_ENV)
self._uihost = os.environ.get(KF_PIPELINES_UI_ENDPOINT_ENV, host)
config = self._load_config(host, client_id, namespace)
```

Reply:
done